support-frontend/assets/pages/paper-subscription-landing/components/NewspaperRatePlanCard.tsx

```diff
@@ -82,8 +82,11 @@ function NewspaperRatePlanCard({
 	);
 
 	return (
-		<div ref={setElementToObserve} css={[card, label && cardWithLabel]}>
-			{label && <div css={cardLabel}>{label}</div>}
+		<div
+			ref={setElementToObserve}
+			css={[card, showBestDealLabel && cardWithLabel]}
+		>
+			{showBestDealLabel && <div css={cardLabel}>Most popular</div>}
```

Comment:
Is it slightly awkward that the prop is called `showBestDealLabel` but the copy is `Most popular`? 🤔  I wonder if there's a general name which would work for both, `showSpecialLabel` maybe?